### ISSUE TITLE
🦄 Improve commit messages 🦄

### ISSUE THREAD

**Issue by husio** [iov-one/weave#463]
To improve our commit messages, I am proposing to include emoji in our messages. There are several projects standarizing this practice. I suggest considering [gitmoji](https://github.com/carloscuesta/gitmoji/) and [commit-message-emoji](https://github.com/dannyfritz/commit-message-emoji) projects.

> [Gitmoji](https://gitmoji.carloscuesta.me) is an initiative to standardize and explain **the use of emojis on GitHub commit messages**.
> 
> **Using emojis** on **commit messages** provides an **easy way** of **identifying the purpose or intention of a commit** with only looking at the emojis used. As there are a lot of different emojis I found the need of creating a guide that can help to use emojis easier.

## Examples

An old and boring commit  

> Fixing ID counter in the decorator. Changes were propagated to our CLI tools as well. Doucmentation was updated.

will be enriched with fun to work with metadata that makes reading enjoyable!

> :bug: :wrench: :books: 
> Fixing ID counter in the decorator. Changes were propagated to our CLI tools as well. Doucmentation was updated.

Happy [Monday! :tada:](https://en.wikipedia.org/wiki/April_Fools%27_Day)

**Comment by ruseinov** [comment]
To be honest I'd really love to keep using git command-line to commit stuff, like I did for the past 10 years, without having to use any macros for emoji. 

The fact that "some projects standardise this" does not mean we necessarily have to, to me it just feels like more work in a place there shouldn't be any. And as someone who never understood people that read comics and such - I find emoji generally redundant. 

I did notice that some of the people on the team like to use this for pretty much anything, but personally I take things way less serious with this approach. 

I'd like to propose limiting the use to chat and tickets and leaving commit messages alone, if possible. Or at least not enforcing this in any way.
